README.md

```diff
@@ -88,6 +88,11 @@ The name of the status created when `create_status` is set to `true`. Defaults t
 Message to comment with on valid release PRs. Can be overridden to add custom
 cheerfulness or reminders. Defaults to `Valid release 🍌`.
 
+## `generate_patch_notes`
+
+If `true`, will use Github's patch note generator instead of the description of the pull-request for the body of the release.
+Generates draft releases instead, to allow for revision before publishing. Also posts the patch notes as a comment of the PR after validation.
```

Comment:
I'll look into solutions to this problem, I hadn't expected releases to generate a lot of discussion. I'm not sure if editing the body of a PR you didn't write yourself is possible; perhaps it is with high enough permissions, but I'm unsure if that's a sound way of solving this. Instead, perhaps posting one comment, and then updating that in-place as the release changes would work? I'll try both, see what works.

Would it be worth removing the pings for the preview? It might not be difficult to just replace the string "@" with "@ ", or even just "\@".

Reply:
Turns out, it's very trivial to change the body, thanks for the suggestion! This is a much more elegant solution now, where the rest of the code is completely unaware of the feature, and all logic happens inside of the generateReleaseNotes function (and one if-statement in action()).